docs/source/gbd2019_models/causes/heart_failure/index.rst

```diff
@@ -152,7 +152,7 @@ pooled cohort analysis_. LDL-C level is assumed not to affect heart failure in t
 
 .. _analysis: https://www.jacc.org/doi/full/10.1016/j.jacc.2019.03.529
 
-The relative risks can be utilized by SBP/LDL-C group based on these tables: 
+The relative risks can be utilized by SBP group based on these tables: 
```

Comment:
```suggestion
The relative risks can be utilized by SBP group based on this table:
```

Reply:
Or is there still an intention to add a second table? I think not, based on the concept model diagram.